src/api/app/controllers/webui/users/tokens_controller.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable Metrics/ClassLength
```

Comment:
I believe it's better to disable the cops with the [.rubocop_todo.yml](https://github.com/openSUSE/open-build-service/blob/9f99e2263a478c1d4a9a2d736f746ebdde61267c/.rubocop_todo.yml#L2)